README.md

```diff
@@ -96,17 +137,18 @@ To create a new organization or import an existing organization, add the followi
 
 ```typescript
 const organization = new Organization(stack, "Organization", {
-  featureSet: FeatureSet.ALL,
+  featureSet: FeatureSet.ALL, // (default) required later on to enable SCPs, enable AWS services or deletage an adminsitrator account
```

Comment:
(default) required later on to enable SCPs, enable AWS services or delegate an adminsitrator account